libs/webb-ui-components/src/components/BridgeInputs/TokenInput.tsx

```diff
@@ -85,36 +92,55 @@ export const TokenInput = forwardRef<HTMLDivElement, TokenInputComponentProps>(
               )}
 
               {token.tokenComposition ? (
-                <Typography component="span" variant="body1" fw="bold">
+                <Typography
+                  component="span"
+                  variant="h5"
+                  fw="bold"
+                  className="capitalize"
+                >
                   {token.tokenComposition[0].trim() +
                     '/' +
                     token.tokenComposition[1].trim()}
                 </Typography>
               ) : (
-                <Typography component="span" variant="body1" fw="bold">
+                <Typography
+                  component="span"
+                  variant="h5"
+                  fw="bold"
+                  className="capitalize"
+                >
                   {token.symbol.trim()}
                 </Typography>
               )}
             </div>
           ) : (
-            <Typography variant="body1" fw="bold">
-              Select {title.toLowerCase()}
+            <Typography variant="h5" fw="bold">
+              Select token
```

Comment:
I'm curious as to why you removed the `{title.toLowerCase()}` here.

Reply:
To maintain consistency, if a token is not selected, all token inputs will now display the same placeholder text - `Select token`